src/core/net_db.cc

```diff
@@ -387,16 +387,11 @@ void NetDb::SaveUpdated() {
       if (it.second->IsUnreachable()) {
         total--;
         // delete RI file
-        if (boost::filesystem::exists(
-              GetFilePath(
-                  full_directory,
-                  it.second.get()))) {
-          boost::filesystem::remove(
-              GetFilePath(
-                  full_directory,
-                  it.second.get()));
-          deleted_count++;
-        }
+        bool is_removed =
+	  boost::filesystem::remove(
```

Comment:
If you're using vim, you can add `autocmd FileType cpp setlocal expandtab shiftwidth=2 softtabstop=2` to your `.vimrc` to help with little things like this. Single spaces in newlines after assigments are discouraged.